frontend/template-partials/translations/src/en/errors.json

```diff
@@ -11,6 +11,14 @@
     "title": "Page not found",
     "description": "This page does not exist"
   },
+  "500": {
+    "title": "Internal Server Error",
+    "description": "Server is down",
+    "header": "Sorry, there is a problem with the service",
+    "paragraph1": "Try again later.",
+    "paragraph2": "We saved your answers. They will be available for 30 days.",
```

Comment:
This isn't true for most of our services

Reply:
ATM there's no content specified for the ticket, i have just used the default GDS template and tweak some section. Please advice what generic content can we use